src/client/components/ui/Form/Combobox/ComboboxMetiers/ComboboxMetiers.test.tsx

```diff
@@ -0,0 +1,391 @@
+
+		const message = screen.getByRole('status');
+		expect(message).toBeVisible();
+		expect(message).toHaveTextContent('Chargement ...');
+	});
+
+	it('affiche un message d’erreur quand le champ est en erreur', async () => {
+		const user = userEvent.setup();
+		const metierServiceMock = aMetierService([]);
+		render(
+			<DependenciesProvider metierService={metierServiceMock}>
+				<ComboboxMetiers
+					name='métier'
+					label='Rechercher un métier'
+					debounceTimeout={0}
+				/>
+			</DependenciesProvider>,
+		);
+
+		await user.type(screen.getByRole('combobox'), 'test');
+		await user.tab();
+
+		const erreur = screen.getByText('Veuillez sélectionner une option dans la liste');
+		expect(erreur).toBeVisible();
+		const combobox = screen.getByRole('combobox');
+		expect(combobox).toHaveAccessibleDescription('Veuillez sélectionner une option dans la liste');
+	});
+
+	it('affiche un message dans les suggestions quand le champ est vide et que l’utilisateur déplie les suggestions', async () => {
+		const user = userEvent.setup();
+		const metierServiceMock = aMetierService();
+		render(
+			<DependenciesProvider metierService={metierServiceMock}>
+				<ComboboxMetiers
+					name='métier'
+					label='Rechercher un métier'
+					debounceTimeout={0}
+				/>
+			</DependenciesProvider>,
+		);
+
+		const deplierSuggestions = screen.getByRole('button', { name: 'Rechercher un métier' });
+		await user.click(deplierSuggestions);
+
+		const message = screen.getByRole('status');
+		expect(message).toBeVisible();
+		expect(message).toHaveTextContent('Commencez à taper pour rechercher un métier');
+	});
+
+	describe('quand le champ est vidé', () => {
+		it('affiche un message dans les suggestions', async () => {
+			const user = userEvent.setup();
+			const metierServiceMock = aMetierService([
+				{ label: 'Génie électrique', romes: ['H1209', 'H1504'] },
+			]);
+			render(
+				<DependenciesProvider metierService={metierServiceMock}>
+					<ComboboxMetiers
+						name='métier'
+						label='Rechercher un métier'
+						debounceTimeout={0}
+					/>
+				</DependenciesProvider>,
+			);
+
+			const combobox = screen.getByRole('combobox');
+			await user.type(combobox, 'Test');
+			await user.clear(combobox);
+
+			const message = screen.getByRole('status');
+			expect(message).toBeVisible();
+			expect(message).toHaveTextContent('Commencez à taper pour rechercher un métier');
+		});
+
+		it('n’affiche aucune suggestion', async () => {
+			const user = userEvent.setup();
+			const metierServiceMock = aMetierService([
+				{ label: 'Génie électrique', romes: ['H1209', 'H1504'] },
+			]);
+			render(
+				<DependenciesProvider metierService={metierServiceMock}>
+					<ComboboxMetiers
+						name='métier'
+						label='Rechercher un métier'
+						debounceTimeout={0}
+					/>
+				</DependenciesProvider>,
+			);
+
+			const combobox = screen.getByRole('combobox');
+			await user.type(combobox, 'Test');
+			await user.clear(combobox);
```

Comment:
c'est pas plutot un 'expect(screen.queryByRole('option')).not.toBeInTheDocument();'